src/LinuxTracingIntegrationTests/IntegrationTestPuppet.cpp

```diff
@@ -194,7 +198,11 @@ int IntegrationTestPuppetMain() {
     } else if (command == PuppetConstants::kDlopenCommand) {
       LoadSoWithDlopenAndCallFunction();
     } else if (command == PuppetConstants::kVulkanTutorialCommand) {
+#ifdef VULKAN_ENABLED
       RunVulkanTutorial();
+#else
+      ORBIT_ERROR("Vulkan isn't enabled");
```

Comment:
Nit: Can you extend the comment to include the flag? Something like: "Compile with WITH_VULKAN".

Reply:
Thanks, done.